data-es6.js

```diff
@@ -18316,7 +18319,7 @@ exports.tests = [
         var desc = Object.getOwnPropertyDescriptor(Object.prototype,"__proto__");
         var A = function(){};
 
-        return (desc
+        return (typeof desc === "object"
```

Comment:
again, this check needs to remain unchanged, because this change allows `null` while the previous line does not (i'm aware the `in` check will throw on null)
```suggestion
        return (desc
```

Reply:
If it remains unchanged, the test result will be undefined as deno deleted Object.prototype.__proto__

Reply:
Then it should fail this test, or, we should be changing the test separately.